components/gfx/display_list/mod.rs

```diff
@@ -76,6 +76,51 @@ impl OpaqueNode {
     }
 }
 
+/// LayerInfo is used to store information PaintLayer metadata during DisplayList construction.
```

Comment:
nit: grammar seems wrong—"LayerInfo is used to store information related to PaintLayer metadata during DisplayList construction", perhaps?